routes/api/v1/favorites.js

```diff
@@ -68,4 +72,51 @@ router.delete('/', (request, response) => {
     }
 });
 
+router.get('/', (request, response) => {
+    if (request.body.api_key) {
+    database('users').where('api_key', request.body.api_key).first()
+      .then((user) => {
+        if (user) {
+
+          database('favorites').where('user_id', user.id)
+            .then(favoriteLocations => {
+              if (favoriteLocations.length) {
+
+                const results = favoriteLocations.map( async (location) => {
+                    const forecastResults = await getForecasts(location, await getLatAndLongFromGoogle(location.location));
+                    return forecastResults;
+                });
+
+                Promise.all(results)
+                  .then(data => {
+                    console.log(data);
+                    response.status(200).json(data);
+                  });
+
+          } else {
+                response.status(200).json({ message: 'No favorites found.' });
+              }
+            })
+            .catch(error => {
+              response.status(500).json(error); // Sometimes gets kicked down here and I dont know why?
+            });
+        } else if (!user) {
+          response.status(401).json({error: 'Unauthorized!'});
+        }
+      }).catch(error => console.log(error));
```

Comment:
Do you want to be console.log (ing) the error? 
I have have found that error is an object and to get anything from it you need to call error.name or error.message

Reply:
You can console the whole log object and it will still show you the error. Console logging the error.messages just takes the most user readable part of the error and sends it back. You can do both but you are right, sending the error.message is a cleaner option.

Reply:
Oh thank you I did not know that!